core/bin/external_node/src/config/mod.rs

```diff
@@ -1447,6 +1467,14 @@ impl From<&ExternalNodeConfig> for TxSenderConfig {
             chain_id: config.required.l2_chain_id,
             // Does not matter for EN.
             whitelisted_tokens_for_aa: Default::default(),
+            timestamp_asserter_params: config.remote.l2_timestamp_asserter_addr.map(|address| {
+                TimestampAsserterParams {
+                    address,
+                    min_time_till_end: Duration::from_secs(
+                        config.optional.timestamp_asserter_min_time_till_end_sec as u64,
```

Comment:
Nit: Since `u32` -> `u64` conversion is lossless, you may use `timestamp_asserter_min_time_till_end_sec.into()` to signal it. Or change the field type to `u64`.